app/src/main/java/io/intrepid/contest/screens/contestcreation/categorieslist/CategoriesListPresenter.java

```diff
@@ -19,6 +19,7 @@
 
     @Override
     public void displayCategories(Contest contest) {
+        if(view == null) return;
```

Comment:
This method gets called from the fragment's onViewCreated() method but interestingly enough on some back and forth swipes, the view comes up as null.